api/layer/neofs.go

```diff
@@ -148,7 +148,7 @@ type NeoFS interface {
 	//
 	// It returns exactly one non-nil value. It returns any error encountered which
```

Comment:
```suggestion
	// It returns exactly one non-zero value. It returns any error encountered which
```

Same for `CreateObject`.